From: Michael Olbrich <m.olbrich@pengutronix.de>
Date: Wed, 6 Mar 2024 08:44:02 +0100
Subject: [PATCH] HACK: don't require GitPython

It's only used to create dist packages, not to build from source
archive. So remove the 'import' to avoid the dependency.

Signed-off-by: Michael Olbrich <m.olbrich@pengutronix.de>
---
 site_scons/site_tools/distsrc.py | 1 -
 1 file changed, 1 deletion(-)

diff --git a/site_scons/site_tools/distsrc.py b/site_scons/site_tools/distsrc.py
index 893552e6ace4..556c7f0790bb 100644
--- a/site_scons/site_tools/distsrc.py
+++ b/site_scons/site_tools/distsrc.py
@@ -20,7 +20,6 @@
 # WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 #
 
-import git
 import SCons
 import os
 import os.path as ospath
